internal/pkg/cli/svc_init.go

```diff
@@ -92,10 +99,12 @@ type initSvcOpts struct {
 	prompt       prompter
 	dockerEngine dockerEngine
 	sel          dockerfileSelector
+	topicSel     topicSelector
 
 	// Outputs stored on successful actions.
 	manifestPath string
 	platform     *string
+	topics       *[]manifest.TopicSubscription
```

Comment:
Why do we need a pointer to a slice? instead of 
```suggestion
	topics       []manifest.TopicSubscription
```